src/main/java/reposense/authorship/model/FileResult.java

```diff
@@ -28,6 +31,10 @@ public String getPath() {
         return path;
     }
 
+    public String getFileType() {
+        return fileType;
+    }
```

Comment:
Same as Eugene's [comment](https://github.com/reposense/RepoSense/pull/787/files#r309656953).

Reply:
The `fileType` in `FileResult` has to be String in this case (line 15). Because `FileResult` will be directly serialized into JSON based on the stricture of `FileResult`.

![image](https://user-images.githubusercontent.com/35756209/62344971-e9f25880-b522-11e9-8458-ddc7016a59fa.png)

This will be the resulting json (on the left side) if we were to change the data type to `FileType`, which will contain information that is unnecessary for the front end.

Reply:
@jylee-git can you try this?

https://stackoverflow.com/questions/53318844/is-it-possible-to-serialize-object-to-json-using-tostring-method-without-custom

Reply:
That link provides solution for jackson serializer, not gson (the one we are using now).

But I'll check if there's any of sorts for gson

Reply:
There's a similar solution for Gson. Will use customized serializer instead